Fix user ID always default on every shell session. (#5117)
RasaHQ/rasa: changelog/5117.improvement.rst

> New user ID will be generated for each shell session.

Reviewer: ```suggestion
New conversation ID will be generated for each shell session.
```

> New command-line argument --conversation-id will be added and wiil give the ability to

Reviewer: ```suggestion
New command-line argument ``--conversation-id`` will be added to ``rasa shell``
```

> Previously the user ID was always 'default'.

Reviewer: ```suggestion
Previously the conversation ID was always ``default``.
```

> set custom conversation ID for each shell session.

Reviewer: ```suggestion
and will give the ability to set custom conversation IDs for each shell session.
```
